src/ngraph/pass/constant_folding_binary.cpp

```diff
@@ -561,20 +535,20 @@ shared_ptr<op::Constant> fold_constant_binary_helper(shared_ptr<op::Constant> a,
 
 bool is_supported_binary_op(std::shared_ptr<Node> n)
 {
-    return (
-        is_type<op::v0::Add>(n) || is_type<op::v1::Add>(n) || is_type<op::v0::Multiply>(n) ||
-        is_type<op::v1::Multiply>(n) || is_type<op::v0::Divide>(n) || is_type<op::v1::Divide>(n) ||
-        is_type<op::v0::Power>(n) || is_type<op::v1::Power>(n) || is_type<op::v0::Equal>(n) ||
-        is_type<op::v1::Equal>(n) || is_type<op::v0::NotEqual>(n) || is_type<op::v1::NotEqual>(n) ||
-        is_type<op::v0::Greater>(n) || is_type<op::v1::Greater>(n) ||
-        is_type<op::v0::GreaterEq>(n) || is_type<op::v1::GreaterEqual>(n) ||
-        is_type<op::v0::Less>(n) || is_type<op::v1::Less>(n) || is_type<op::v0::LessEq>(n) ||
-        is_type<op::v1::LessEqual>(n) || is_type<op::v0::Maximum>(n) ||
-        is_type<op::v1::Maximum>(n) || is_type<op::v0::Minimum>(n) || is_type<op::v1::Minimum>(n) ||
-        is_type<op::v0::And>(n) || is_type<op::v1::LogicalAnd>(n) || is_type<op::v0::Or>(n) ||
-        is_type<op::v1::LogicalOr>(n) || is_type<op::v0::Xor>(n) ||
-        is_type<op::v1::LogicalXor>(n) || is_type<op::v0::Subtract>(n) ||
-        is_type<op::v1::Subtract>(n));
+    return (is_type<op::v0::Multiply>(n) || is_type<op::v1::Multiply>(n) ||
+            is_type<op::v0::Divide>(n) || is_type<op::v1::Divide>(n) || is_type<op::v0::Power>(n) ||
+            is_type<op::v1::Power>(n) || is_type<op::v0::Equal>(n) || is_type<op::v1::Equal>(n) ||
+            is_type<op::v0::NotEqual>(n) || is_type<op::v1::NotEqual>(n) ||
+            is_type<op::v0::Greater>(n) || is_type<op::v1::Greater>(n) ||
+            is_type<op::v0::GreaterEq>(n) || is_type<op::v1::GreaterEqual>(n) ||
+            is_type<op::v0::Less>(n) || is_type<op::v1::Less>(n) || is_type<op::v0::LessEq>(n) ||
+            is_type<op::v1::LessEqual>(n) || is_type<op::v0::Maximum>(n) ||
+            is_type<op::v1::Maximum>(n) || is_type<op::v0::Minimum>(n) ||
+            is_type<op::v1::Minimum>(n) || is_type<op::v0::And>(n) ||
+            is_type<op::v1::LogicalAnd>(n) || is_type<op::v0::Or>(n) ||
+            is_type<op::v1::LogicalOr>(n) || is_type<op::v0::Xor>(n) ||
+            is_type<op::v1::LogicalXor>(n) || is_type<op::v0::Subtract>(n) ||
+            is_type<op::v1::Subtract>(n));
```

Comment:
Just a note, I understand this was already in the code: Why do we need this function? We will face the same list when will do real constant folding in `fold_constant_binary_arithmetic`.

Reply:
I wouldn't have written it this way, but it is what it is until it's replace with the new way.